ocaml/xapi-consts/constants.ml

```diff
@@ -152,6 +152,13 @@ let audit_log_uri = "/audit_log"
 let get_pool_update_download_uri = "/update/"
 
 (* ocaml/xapi/xapi_pool_update.ml *)
+
+let get_repository_uri = "/repository/" (* ocaml/xapi/repository.ml *)
```

Comment:
Does this really depend on the trailing `/`? Is there a difference in an URI between `a/b` and `a/b/`? If not, I would prefer to drop the trailing slash.